combat-trainer.lic

```diff
@@ -2183,6 +2193,33 @@ class GameState
     end
   end
 
+  def tactics_analyze?
```

Comment:
There seems to be a lot of overlap with the `perform_analyze?` method. Aren't they basically doing the same thing?